internal/fs/inode/core.go

```diff
@@ -32,6 +32,25 @@ var (
 	NonexistentType Type = 5
 )
 
+func (t Type) String() string {
```

Comment:
we can remove this if we don't add debug logs for type cache ?

Reply:
No, it also helps during debugging using and IDE or even using Delve, by displaying the name instead of 0,1,2,3 etc.